bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/SamsungTvTlsTrustManagerProvider.java

```diff
@@ -18,35 +18,52 @@
 
 import javax.net.ssl.SSLEngine;
 import javax.net.ssl.X509ExtendedTrustManager;
+import javax.security.auth.x500.X500Principal;
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
 import org.eclipse.smarthome.io.net.http.TlsTrustManagerProvider;
 import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * Provides a TrustManager to allow secure websocket connections to any TV (=server)
+ * TODO: convert to using TrustAllTrustMananger (not supported in OH2.4 yet)
  *
  * @author Arjan Mels - Initial Contribution
  */
 @Component
 @NonNullByDefault
 public class SamsungTvTlsTrustManagerProvider implements TlsTrustManagerProvider {
+    private final static Logger logger = LoggerFactory.getLogger(WakeOnLanUtility.class);

```

Comment:
Do we need `static`? In general we try to avoid static loggers.

Reply:
Sorry, no need, didn't get your intention the first time.